pkg/cmd/kind/delete/clusters/deleteclusters.go

```diff
@@ -48,7 +48,7 @@ func NewCommand(logger log.Logger, streams cmd.IOStreams) *cobra.Command {
 		},
 	}
 	cmd.Flags().StringVar(&flags.Kubeconfig, "kubeconfig", "", "sets kubeconfig path instead of $KUBECONFIG or $HOME/.kube/config")
-	cmd.Flags().BoolVar(&flags.All, "all", false, "delete all clusters")
+	cmd.Flags().BoolVar(&flags.All, "all", false, "delete all kind clusters associated with a kubeconfig")
```

Comment:
I think that's even more confusing.
it will delete all kind clusters, just with the asterisk that if you:
`kind export kubeconfig` to some random paths
OR
`kind create cluster --kubeconfig=/some-random-path`

then `kind delete clusters` will not be touching those files, we don't centrally track this.

whereas in the simpler world of `kind create cluster` pre 0.6 `kind delete cluster` was guaranteed to remove the file.

given the `kind export kubeconfig` case, I think maybe we can just close https://github.com/kubernetes-sigs/kind/issues/1260 as WAI

if you set `--kubeconfig` on this command it will remove the matching clusters in that file.

you can leverage this to do `kind delete clusters --all` repeatedly with different `--kubeconfig` to clean all of them (and only the first time remove containers unless more are created)

Reply:
Sure, that's what I originally had so I'll just change it back to remove `kind clusters`. 

I can link to your post comment here and close #1260. If someone has a better approach down the road, at least thoughts around it are documented in the closed issue.